packages/cli/src/start/serve.js

```diff
@@ -9,10 +10,27 @@ const client = require("@patternplate/client");
 module.exports = patternplate;
 
 async function patternplate(options) {
-  const { port } = options;
+  const { port, listen,
+    privateKey, certChain,
+    hostName, httpsRedirectPort } = options;
+
+  if (httpsRedirectPort > 0) {
+    const redirectHttp = express();
```

Comment:
`.on("error")` on `redirectHttp` should be handled / cause the startup to fail, especially for cases where the port is already taken